sources/niScript/tsrc_ni/tu_linter.nip

```diff
@@ -1,6 +1,74 @@
+//
```

Comment:
tu_linter gives a good overview of all the linting rules as there should be a test for every linting rule introduced.